unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -1,13 +1,16 @@
-using DCL;
-using DCL.Helpers;
-using DCL.Interface;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using DCL;
+using DCL.Helpers;
+using DCL.Interface;
+using DCL.NotificationModel;
 using UnityEngine;
 using UnityEngine.Rendering;
 using UnityEngine.Rendering.Universal;
-using Categories = WearableLiterals.Categories;
```

Comment:
This was helping to reduce the boilerplate in the class, were we losing too much clarity?

Reply:
yeah i agree, the IDE automatically changed it when optimized imports